bitnami/elasticsearch/values.yaml

```diff
@@ -130,6 +130,35 @@ discovery:
 #     mountPath: /certs
 #     readOnly: true
 
+## initdb scripts
+## Specify dictionary of scripts to be run at first boot
+## Alternatively, you can put your scripts under the files/docker-entrypoint-initdb.d directory
+##
+# initdbScripts:
+#   my_init_script.sh: |
+#      #!/bin/sh
+#      echo "Do something."
+#
+## ConfigMap with scripts to be run at first boot
+## Note: This will override initdbScripts
+# initdbScriptsConfigMap:
```

Comment:
I would leave it uncommented but empty. It is very useful if you want to use automation tools for detecting if values are defined in the template or in the README.

Reply:
For example, you are doing it with the values below

Reply:
OK.